### PR TITLE
Update Breakpoints state only once when "Toggle all breakpoints"

### PR DESCRIPTION
**Pull request by nyrosmith**
Associated Issue: #5563

### CODE REVIEW

**Comment by jasonLaster** [comment]
First pass looks good!

File: src/actions/types.js [hunk 1]
```diff
@@ -125,6 +125,8 @@ type BreakpointAction =
       value: AddBreakpointResult
     }
   | { type: "DISABLE_BREAKPOINT", breakpoint: Breakpoint }
+  | { type: "DISABLE_ALL_BREAKPOINTS", breakpoints: Breakpoint[] }
+  | { type: "ENABLE_ALL_BREAKPOINTS", breakpointS: Breakpoint[] }
```
Comment (nyrosmith) [inline src/actions/types.js]: Typo breakpointS

**Comment by jasonLaster** [comment]
@nyrosmith mind rebasing?

**Comment by jasonLaster** [comment]
seems relevant

**Pull request merged**